test/main/accounts/index.test.js

```diff
@@ -695,3 +698,52 @@ describe('#removeRequests', () => {
     expect(Accounts.removeRequest).not.toHaveBeenCalled()
   })
 })
+
+describe('#signerCompatibility', () => {
+  beforeEach(() => {
+    store.navDash = jest.fn()
+  })
+
+  afterEach(() => {
+    store.removeSigner('12')
+  })
+
+  const signerTypes = ['trezor', 'ledger', 'lattice', 'ring', 'seed', 'aragon']
+  
+  signerTypes.forEach((signerType) => {
+    it(`should open the signer menu when a ${signerType} signer is not available`, (done) => {
+      Accounts.addRequest(request)
+      signers.get.mockReturnValue('12')
```

Comment:
this method actually returns a signer, not an id. for clarity of what the test is doing we can just use the signer we add to the store below

Reply:
For clarity of what the tests are doing, we should rewrite the whole file 😅

Reply:
but yes.